python/pyarrow/rados.py

```diff
@@ -38,6 +59,14 @@ def round(self, num):
         return int(result_str)
 
     def write_file(self, filename, table):
+        """
+        The call to write a table to a file.
+
+        Parameters
+        ---------
+        filename: Name of the file to write.
+        table: The table to write.
```

Comment:
We can say "The table slice to write". This would provide more clarity to the reader

Reply:
Done.